Makefile

```diff
@@ -64,7 +64,6 @@ install-openvino-test:
 	pip install -U pip
 	pip install -e .
 	pip install "git+https://github.com/openvinotoolkit/open_model_zoo.git@37f60eb#egg=accuracy_checker&subdirectory=tools/accuracy_checker"
-	pip install tensorflow==2.12.0 # Install tensorflow before to avoid conflict on install for typing-extensions
```

Comment:
Have you checked that `install-openvino-test` make command uses only for pre-commit tests and does not affect other internal jobs?

Reply:
Thank you for the explanation. 
My question was about the consequences of this change. Are there any jobs (including internal runs) that utilize `install-openvino-test` make command that may fail without TF?

Reply:
As i have wrote i dont found any code that required TF. As owner of OV part may be you know more that me.
Please provide list of jobs that may fail.